portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,46 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 90%;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
 }
+
+a:link {
+  color: black;
+}
+
+h1 {
+  margin-left: auto;
+  margin-right: auto;
+  font: 4.5em Arial, Helvetica, sans-serif;
+}
+
+a {
+  font: 1.5em Verdana, Geneva, Tahoma, sans-serif;
+}
+
+p{
+  font: 1.0em Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.upper-info {
+  color: gray;
+}
+
+.header-image {
+  position: absolute;
+  top: 10px;
+  right: 50%;
+  width:15%; 
```

Comment:
Minor style thing here too, add a space after the colon here and also the width property in the next rule below.

Reply:
Not sure how I missed this and the space in '<p >' earlier, I find inconsistent spacing irritating if I catch them haha. Thanks for your help and thorough suggestions!